lib/core/src/lib/datatable/components/amount-cell/amount-cell.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="value$ | async as amount">
```

Comment:
How this solution handle wrong values like a objects? The angular pipe is prepared for strings&numbers only.

Reply:
this is default error behaviour - display nothing
example:
(first row has incorrect value of car price)
<img width="863" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/49343696/98726a8f-5b9c-4321-b4ca-9c285838c231">